playwright.config.ts

```diff
@@ -68,51 +68,103 @@ const config: PlaywrightTestConfig<TestOptions> = {
       testMatch: "login.spec.ts",
     },
     {
-      name: "chromium:lxd-5.0-edge",
+      name: "chromium:lxd-5.0-edge:unclustered",
       use: {
         ...devices["Desktop Chrome"],
         lxdVersion: "5.0-edge",
       },
     },
     {
-      name: "firefox:lxd-5.0-edge",
+      name: "firefox:lxd-5.0-edge:unclustered",
       use: {
         ...devices["Desktop Firefox"],
         lxdVersion: "5.0-edge",
       },
     },
     {
-      name: "chromium:lxd-5.21-edge",
+      name: "chromium:lxd-5.21-edge:unclustered",
       use: {
         ...devices["Desktop Chrome"],
         lxdVersion: "5.21-edge",
       },
       dependencies: ["login-chromium"],
     },
     {
-      name: "firefox:lxd-5.21-edge",
+      name: "firefox:lxd-5.21-edge:unclustered",
       use: {
         ...devices["Desktop Firefox"],
         lxdVersion: "5.21-edge",
       },
       dependencies: ["login-firefox"],
     },
     {
-      name: "chromium:lxd-latest-edge",
+      name: "chromium:lxd-latest-edge:unclustered",
       use: {
         ...devices["Desktop Chrome"],
         lxdVersion: "latest-edge",
       },
       dependencies: ["login-chromium"],
     },
     {
-      name: "firefox:lxd-latest-edge",
+      name: "firefox:lxd-latest-edge:unclustered",
       use: {
         ...devices["Desktop Firefox"],
         lxdVersion: "latest-edge",
       },
       dependencies: ["login-firefox"],
     },
+    {
+      name: "chromium:lxd-5.0-edge:clustered",
+      use: {
+        ...devices["Desktop Chrome"],
+        lxdVersion: "5.0-edge",
+      },
+      testMatch: "*-clustered.spec.ts",
+    },
```

Comment:
```suggestion
    {
      name: "chromium:lxd-5.0-edge:clustered",
      use: {
        ...devices["Desktop Chrome"],
        lxdVersion: "5.0-edge",
      },
      testMatch: "*-clustered.spec.ts",
      dependencies: ["enable-clusterinng-chrome"],
    },
```
We need to enable clustering before running the new `:clustered` suites. So maybe add another suite similar to login tests just to click the enable clustering button?